content/_data/authors/dheerajodha.adoc

```diff
@@ -3,4 +3,4 @@ name: "Dheeraj Singh Jodha"
 github: dheerajodha
 linkedin: dheeraj-sj
 ---
-Dheeraj is working as an Associate Software Quality Engineer at Red Hat. He is the Jenkins Google Summer of Code (GSoC) contributor in 2022, associated with the link:https://github.com/jenkins-infra/plugin-health-scoring[Plugin Health Scoring] project. He started his journey of contributing to Jenkins in March 2021. He is part of the Jenkins Documentation SIG and has also made some contributions to the link:https://github.com/jenkinsci/custom-distribution-service[Custom Distribution Service for Jenkins] project.
+Dheeraj is working as an Associate Software Quality Engineer at Red Hat. He is a Jenkins Google Summer of Code (GSoC) contributor in 2022, associated with the link:https://github.com/jenkins-infra/plugin-health-scoring[Plugin Health Scoring] project. He started his journey of contributing to Jenkins in March 2021. He is part of the Jenkins Documentation SIG and has also made some contributions to the link:https://github.com/jenkinsci/custom-distribution-service[Custom Distribution Service for Jenkins] project.
```

Comment:
```suggestion
Dheeraj is working as an Associate Software Quality Engineer at Red Hat.
He is a Jenkins Google Summer of Code (GSoC) contributor in 2022, associated with the link:https://github.com/jenkins-infra/plugin-health-scoring[Plugin Health Scoring] project.
He started his journey of contributing to Jenkins in March 2021.
He is part of the Jenkins Documentation SIG and has also made some contributions to the link:https://github.com/jenkinsci/custom-distribution-service[Custom Distribution Service for Jenkins] project.
```
It helps with review if each sentence is its own line in the document. The page will format the text in line if there are no empty lines between sentences. If you try formatting it like that, and doing a `make run` on your local clone, it should appear as intended.

Reply:
Sounds good, I'll keep this in mind. Thank you for letting me know.